sourcegraph/sourcegraph
# Setup Wizard: Mobile response styling

Closes #47435, adds mobile responsiveness to 3 steps in Setup Wizard

## Test plan
- Run from app to see all steps (`sg start app`)
- Test flow on all screen sizes (down to 320px screen width)
- Ensure no breaking UI

## App preview:

- [Web](https://sg-web-becca-setup-wiz-mobile-styling.onrender.com/search)

Check out the [client app preview documentation](https://docs.sourcegraph.com/dev/how-to/client_pr_previews) to learn more.
